ecosystem/gstreamer_plugin/gstmtltxsink.c

```diff
@@ -185,6 +255,53 @@ static gboolean gst_mtltxsink_parse_fps(GstVideoInfo* info, enum st_fps* fps) {
   return TRUE;
 }
 
+static gboolean gst_mtltxsink_parse_fps_code(gint fps_code, enum st_fps* fps) {
+
+  if (!fps) {
+    GST_ERROR("Invalid fps pointer");
+    return FALSE;
+  }
+
+  switch (fps_code) {
```

Comment:
Is it worthy to sort the cases in some way, the best would be from smallest to largest. Unless this order is basing on the probability of usage.

Reply:
i think its good idea to leave draft PR without reviews ^^
I appreciate the effort though